core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -55,7 +55,7 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
     overridingProps.put(ServerLogConfigs.NUM_PARTITIONS_CONFIG, 4.toString)
     TestUtils.createBrokerConfigs(
       numServers,
-      zkConnectOrNull,
+      null,
```

Comment:
Could we even remove the argument from `createBrokerConfigs()`?

Reply:
The `createBrokerConfigs()` will modify about 40 files, I will addressed it on another PR and open another Jira to trace it.

Reply:
Some tests are still using not `null` value on this method.
https://github.com/apache/kafka/blob/ed10fc63a9fd3032a529f7e10a0f863ee71b710a/core/src/test/scala/unit/kafka/server/ReplicaAlterLogDirsThreadTest.scala#L208

Reply:
That makes sense, then we can do this in another PR. Thanks